src/main/java/com/eternalcode/combat/fight/tagout/FightTagOutController.java

```diff
@@ -0,0 +1,21 @@
+package com.eternalcode.combat.fight.tagout;
+
+import com.eternalcode.combat.fight.event.FightTagEvent;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.EventPriority;
+import org.bukkit.event.Listener;
+
+public class FightTagOutController implements Listener {
+    FightTagOutService tagOutService;
+
+    public FightTagOutController(FightTagOutService tagOutService) {
+        this.tagOutService = tagOutService;
+    }
+
+    @EventHandler(priority = EventPriority.NORMAL)
```

Comment:
EventPriority NORMAL is the default.
```suggestion
    @EventHandler
```